### PR TITLE
improve readme

### PR DESCRIPTION
**Pull request by KuthumiPepple**
@tadayosi @Delawen I added more information to the readme. The examples run correctly. Let me know if there's more to be done.

### CODE REVIEW

**Comment by Delawen** [comment]
LGTM

File: generic-examples/languages/README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
-# Supported languages Camel K examples
+# Supported languages for writing Camel K integrations
 
-Find useful examples about how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML` and `YAML`.
\ No newline at end of file
+Contains useful examples on how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML`, `Kotlin` and `YAML`.
```
Comment (tadayosi) [inline generic-examples/languages/README.md:3]: A bit nitpicky but it should be `JavaScript`

File: generic-examples/languages/README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+Read the general instructions in the [root README.md file](/README.md) for setting up your environment and the Kubernetes cluster before looking at this example.
+
+Make sure you've read the [installation instructions](https://camel.apache.org/camel-k/latest/installation/installation.html) for your specific
+cluster before starting the example.
+
+## Understanding the examples
+- [routes.kts](./routes.kts): defines a route that logs a message every second
+- [routes.yaml](./hello.xml): defines a route that every 5 secs, transforms the message content to uppercase and logs it.
+- [simple.js](./simple.js): logs message periodically, the multiline query parameter is set to true, thus each information is output on a newline
```
Comment (tadayosi) [inline generic-examples/languages/README.md:17]: There are more examples here. Why only these three?

**Review by tadayosi — CHANGES_REQUESTED** [review]
LGTM but some small improvement requests.

File: generic-examples/languages/README.md (hunk 1 above)
Comment (KuthumiPepple) [reply generic-examples/languages/README.md:3]: Done!

File: generic-examples/languages/README.md (hunk 2 above)
Comment (KuthumiPepple) [reply generic-examples/languages/README.md:17]: I felt like the ones I left out were a bit similar to the ones I described. But you are right, we are showing the supported languages, so they should all be described. Done!

**Review by tadayosi — APPROVED** [review]

**Pull request merged**